### PR TITLE
First pull request for the purpose of review

### PR DESCRIPTION
**Pull request by moheed-ali**
This pull request provides the first working version of bike-mobility-example with a waypoint mobility model and periodic sender#

## Proposed Changes

  - Creation of new bike-mobility-helper
  - Creation of new bike-application

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ set(source_files
     model/app/server/network-controller.cc
     model/app/server/network-controller-components.cc
     model/app/server/adr-component.cc
+    model/app/bike-application.cc #new entry
```
Comment (non-det-alle) [inline CMakeLists.txt:12]:
```suggestion
    model/app/bike-application.cc
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -18,7 +19,7 @@ set(source_files
     model/mac/lorawan-mac.cc
     model/mac/gateway-lorawan-mac.cc
     model/mac/base-end-device-lorawan-mac.cc
-    model/mac/class-a-end-device-lorawan-mac.cc
+    model/mac/class-a-end-device-lorawan-mac.cc
```
Comment (non-det-alle) [inline CMakeLists.txt:22]:
```suggestion
    model/mac/class-a-end-device-lorawan-mac.cc
```

**Comment by nondetalle** [comment]
Hello,
I reviewed the code, made some modifications, and added a couple of examples.
I will now merge the branch.
Best,
Alessandro

**Comment by moheed-ali** [comment]
Hello Alessandro,

Thank you for your update!

Regards,
Moheed Ali Kayani
________________________________

**Pull request merged**